client/my-sites/upgrades/domain-management/transfer/utils.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import notices from 'notices';
+import { translate } from 'lib/mixins/i18n'
+
+export const displayRequestTransferCodeResponseNotice = ( responseError, domainStatus ) => {
+	if ( responseError ) {
+		this.displayResponseError( responseError );
```

Comment:
This will fail - `this.displayResponseError` is `undefined` (or actually just `this`, according to Chromium). You should drop `this` and declare `displayResponseError` first in the file.